ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -2886,6 +3050,273 @@
 			};
 			name = Release;
 		};
+		B681E9151D380633001157CE /* Development */ = {
```

Comment:
No. But this is a todo repository and it's modified to adapt to the todo project settings. In the case when we need to send the pull request back against main open source repository, we need to get rid of all these changes.